packages/cfpb-forms/src/organisms/Multiselect.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable max-lines-per-function */
```

Comment:
What was this added for? If it was for `function Multiselect( element ) {` we should only disable it for that line, not the whole file. We'd add `// eslint-disable-line max-statements max-lines-per-function` to line 21.